sbt-version-policy/src/main/scala/sbtversionpolicy/SbtVersionPolicySettings.scala

```diff
@@ -54,6 +53,7 @@ object SbtVersionPolicySettings {
   )
 
   def reconciliationSettings = Def.settings(
+    versionPolicyDependencySchemes := libraryDependencySchemes.value,
```

Comment:
This overrides `ThisBuild / versionPolicyDependencySchemes` setting which means that the user cannot define some `versionPolicyDependencySchemes` for all projects in the build at once.

Also it breaks the compatibility with sbt version below 1.5.

I suggest this instead:
```suggestion
    versionPolicyDependencySchemes += SettingKey[Seq[ModuleID]]("libraryDependencySchemes").?.value.toSeq.flatten,
```

In which case the global `versionPolicyDependencySchemes := Seq.empty` should probably be kept.

Reply:
Good point. I thought sbt-version-policy already required sbt 1.5.x anyway, but it works with sbt 1.4.x. I will document that, and use your trick to not require sbt 1.5.x. Thanks!

Reply:
> This overrides `ThisBuild / versionPolicyDependencySchemes` setting which means that the user cannot define some `versionPolicyDependencySchemes` for all projects in the build at once.

Indeed, but this is on purpose, users should define a global `libraryDependencySchemes`, not `versionPolicyDependencySchemes` (which is deprecated).

Reply:
If you want to deprecate it in favor of `libraryDependencySchemes`, then you should derive `libraryDependencySchemes` from `versionPolicyDependencySchemes` (in a way that preserves both), and *use* `libraryDependencySchemes` later. So this should be:
```scala
libraryDependencySchemes ++= versionPolicyDependencySchemes.value
```
with still keeping the default `versionPolicyDependencySchemes` in `schemesGlobalSettings`.
And then use `libraryDependencySchemes.value` everywhere else instead of `versionPolicyDependencySchemes.value`.

That should preserve the semantics of old builds.

Reply:
> So this should be:
> 
> ```scala
> libraryDependencySchemes ++= versionPolicyDependencySchemes.value
> ```

This is not a good idea IMHO because `libraryDependencySchemes` only supports a subset of what `versionPolicyDependencySchemes` did support. So, although it is possible to derive `versionPolicyDependencySchemes` from `libraryDependencySchemes`, the other direction is not always possible.

Reply:
Hum, but that means that the deprecation warning is misleading. Users cannot always replace `versionPolicyDependencySchemes` by `libraryDependencySchemes`.

What isn't supported any more? What should users do when they hit those?

Reply:
Users can always migrate from vpDependencySchemes to libraryDependencySchemes, it’s just that it might require more work than just:

~~~ diff
- versionPolicyDependencySchemes += "foo" %% "bar" % "semver-spec"
+ libraryDependencySchemes += "foo" %% "bar" % "semver-spec"
~~~

`versionPolicyDependencySchemes` used to support wildcards in module IDs:

~~~ scala
versionPolicyDependencySchemes += "org.typelevel" %% "cats-*" % "semver-spec"
~~~

This is not supported by `libraryDependencySchemes`, so users will have to write something like the following, instead:

~~~ scala
libraryDependencySchemes ++= Seq(
  "org.typelevel" %% "cats-core" % "semver-spec",
  "org.typelevel" %% "cats-effects" % "semver-spec"
)
~~~

Reply:
So basically sbt users will have to move to sbt 1.5 to take advantage of the `versionScheme` declaration in their dependencies.

Reply:
OK, in that case, I suggest to still introduce @adpi2's suggested change, but without the compatibility hack for sbt 1.4.0:
```suggestion
    versionPolicyDependencySchemes ++= libraryDependencySchemes.value,
```
and keep the initialization to `Seq.empty` in `schemesGlobalSettings`. This way, we preserve the semantics of existing builds that set `versionPolicyDependencySchemes` in the `ThisBuild` or `Global` scope.

Otherwise LGTM.

Reply:
OK, done.